content/source/docs/cloud/workspaces/vcs.html.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: "cloud"
+page_title: "VCS Connections - Workspaces - Terraform Cloud"
+---
+
+# Configuring Workspace VCS Connections
+
+Any Terraform Cloud workspace can be connected to a version control system (VCS) repository that contains its Terraform configuration. You can assign a connection [when creating a workspace](./creating.html), and can modify that connection in the "Version Control" page of the workspace's settings.
+
+For more information about how configuration versions and connected repositories work, see [Terraform Configurations in Terraform Cloud Workspaces](./configurations.html).
+
+-> **API:** See the [Update a Workspace endpoint](../api/workspaces.html#update-a-workspace) (`PATCH /organizations/:organization_name/workspaces/:name`).
+
+To change the VCS settings for an existing workspace, choose "Version Control" from the workspace's "Settings" menu. The version control page includes the following settings:
+
+- Which VCS repository to use.
+- Which directories in the repository should trigger Terraform runs when they change.
+- Which branch of the repository to use.
+- Whether to download the repository's submodules.
+
+For most of the settings on this page, you must save any changes with the "Update VCS settings" button at the bottom of the page.
+
+## VCS Connection and Repository
```

Comment:
Is the content in this section duplicated from somewhere else? It feels familiar to me. If so instead of duplicating link out if possible.

Reply:
UNFORTUNATELY, the product is kind of against me here. This interface is VERY similar to the flow on the "create a new workspace" page, but it has just enough critical differences that the descriptions can't be combined.

Reply:
(I tried. 😑)